pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -482,6 +482,30 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: DockerBuildCreatorRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     util.NewStringSet("create"),
+					Resources: util.NewStringSet("builds/docker", "builds/custom"),
```

Comment:
we probably want to add constants for things that other parts of the code base are running subject access reviews on... these, "imagestreams/layers", etc